packages/flutter_tools/lib/src/project.dart

```diff
@@ -671,8 +689,7 @@ $javaGradleCompatUrl
   }
 
   String? get group {
```

Comment:
I know the group behavior is unrelated to this change but would you be willing to add the following documentation to line 462 where we define _groupPattern. 
```
/// Pattern used to find the Gradle "group" string. 
/// Expected example: `group "dev.flutter.plugin"`
/// Regex is used in groovy and kotlin gradle files. 
```

Reply:
good idea, done in f249659a83b52c8861fe0aa9805cb47ffbf9f3c3